core/config/docs/chains-cosmos.toml

```diff
@@ -23,6 +21,10 @@ OCR2CachePollPeriod = '4s' # Default
 OCR2CacheTTL = '1m' # Default
 # TxMsgTimeout is the maximum age for resending transaction before they expire.
 TxMsgTimeout = '10m' # Default
+# Bech32Prefix is the human-readable prefix for addresses on this Cosmos chain. See https://docs.cosmos.network/v0.47/spec/addresses/bech32.
+Bech32Prefix = 'wasm' # Default
+# FeeToken is the token denomination which is being used to pay gas fees on this chain.
+FeeToken = 'ucosm' # Default
```

Comment:
It looks like these Cosmos fields were kept sorted (after the common chain id and enabled fields) - let's keep that consistent.

Reply:
It might be worth doing the same on the type defined in chainlink-cosmos, so that all of the toml is formatted consistently.